Bottom: dcc3afa9708f65b64f13687eb147ea5fc3bbb3de
Top:    867712ea8853622f71c69e8c4c77d7060e806b05
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-11-25 20:10:36 +0000

Refresh of patch

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 59279d0..3ca7387 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -946,7 +946,8 @@ load_built_in_shaders (void)
   char *circular_fs_source =
           "void main()\n"
           "{\n"
-          "  float sqdist = dot (gl_TexCoord[0].st, gl_TexCoord[0].st);\n"
+          "  float sqdist;\n"
+          "  sqdist = dot (gl_TexCoord[0].st, gl_TexCoord[0].st);\n"
           "  if (sqdist > 1.0)\n"
           "    discard;\n"
           "  gl_FragColor = gl_Color;\n"
